src/classes/Accounts.cls

```diff
@@ -0,0 +1,34 @@
+public class Accounts extends fflib_SObjectDomain {
+    public Accounts(List<Account> sObjectList)
+    {
+        super(sObjectList);
+    }
+    
+    public class Constructor implements fflib_SObjectDomain.IConstructable {
+        public fflib_SObjectDomain construct(List<SObject> sObjectList) {
+            return new Accounts(sObjectList);
+        }
+    }
+    
+    public static void onUpdateAccount(fflib_SObjectUnitOfWork uow)
+    {
+        uow.commitWork();
```

Comment:
Could you please explain what this commitWork do here?

Reply:
This commits the unit of work that we registered via unit of work into the database